### ISSUE TITLE
on mobile side support adjusting the brightness 

### ISSUE THREAD

**Issue by icerleer** [MoePlayer/DPlayer#150]
hello,  thanks for your dplayer,  whether to consider adjusting the brightness on the mobile side ?

**Comment by micooz** [comment]
/cc @DIYgod
Do you think it's possible to access system configuration direclty or not-directly via web APIs?

**Comment by DIYgod** [comment]
No, I don't think so, maybe a translucent mask can simulate it.

Anyway, I don't think we need it.

**Comment by donnes** [comment]
Actually it's not supported for various browsers. Only in native way.

**Comment by icerleer** [comment]
@DIYgod  I know what you mean.. 

But there is a problem, when we use dplayer before playing, the system brightness is 50%, then how do we use the transparent mask set to 100%

**Comment by icerleer** [comment]
@DIYgod @micooz @donnes  

I think, if it can be achieved on the mobile side,  dplayer experience will be better

**Comment by donnes** [comment]
@icerleer The point is that not have a way to control native system from browser Javascript APIs.
Is important to know that DPlayer is a browser video player and just have support for mobile browser.
